styles/components/HackathonModal.module.scss

```diff
@@ -0,0 +1,95 @@
+// Import fonts
+@import url('https://fonts.cdnfonts.com/css/motorblock');
+
+.Modal {
+  @apply w-11/12 max-w-2xl outline-none overflow-hidden;
+  @apply bg-opacity-0;
+}
+
+.modalImageContainer {
+  @apply absolute w-52 h-72 z-40;
```

Comment:
Can you set the w or the h we can discuss this more if you want but the css aspect-ratio property is so much better here set it to match the aspect ratio of the image. Also how does this work when the page goes below this width?

This makes it more dynamic and fitting to the image and lets us adjust the width when needed without having to calculate the new height.